README.md

```diff
@@ -65,6 +66,17 @@ GCC>=5 is ABI compatible for minor versions. To solve multiple minors, there are
 | - [conanio/clang7: clang 7](https://hub.docker.com/r/conanio/clang7/)                 | x86_64 |  Supported |
 
 
+#### Conan Server
+
+Conan Docker Tools provides an image version with only Conan Server installed, very useful for the cases it is necessary to run a server without touching the host.
+
+| Version                                                                                       | Arch   |  Status, Life cycle  |
+|-----------------------------------------------------------------------------------------------|--------|------------|
+| - [conanio/conan_server](https://hub.docker.com/r/conanio/conan_server/)             | ANY |  Supported |
+
+#### Conan Installer
+
+**conanio/gcc7-centos6** is a special image version based on CentOS 6, GCC 7 and **glibc 2.12** to build installers and make compatible to ancient glib version.
```

Comment:
**conanio/gcc7-centos6** is a special image version based on CentOS 6, GCC 7 and **glibc 2.12** (very old glibc version). This is intended to build executables that run almost on any Linux because  **glibc** guarantees backward compatibility. You can use this image to build your Conan build tools packages (`build_requires`).

Reply:
done!